src/gui/accountsettings.cpp

```diff
@@ -1745,7 +1745,7 @@ void AccountSettings::setupE2eEncryptionMessage()
 {
     _ui->encryptionMessage->setMessageType(KMessageWidget::Information);
     _ui->encryptionMessage->setText(tr("This account supports end-to-end encryption, but it needs to be set up first."));
-    _ui->encryptionMessage->setIcon(Theme::createColorAwareIcon(QStringLiteral(":/client/theme/black/state-info.svg")));
+    _ui->encryptionMessage->setIcon(Theme::createColorAwareIcon(QStringLiteral(":/client/theme/info.svg")));
```

Comment:
any reason for not using a system icon here? https://doc.qt.io/qt-6/qicon.html#ThemeIcon-enum

```suggestion
    _ui->encryptionMessage->setIcon(QIcon::fromTheme(QIcon::ThemeIcon::DialogInformation));
```

Reply:
from my understanding Qt 6.7+ allows for that to work on macOS + Windows as well https://doc.qt.io/qt-6/qicon.html#creating-an-icon-from-a-theme-or-icon-library

either way, this was just a curiosity from my end.  I'm fine with providing the icon ourselves to stay consistent on all platforms